src/components/media_manager/CMakeLists.txt

```diff
@@ -32,17 +32,20 @@ include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
 include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
 include_directories(
-    include
+    ${COMPONENTS_DIR}/media_manager/include/
```

Comment:
@yang1070 media manager depends from application_manager interface, and application manager depends from functional module interface. With commit 31d41e6935dfceb9c374d67ab44cf5b7b26930a1 I removed dependency of medai manager tests from functional_module library